pkg/logstash.sysv.redhat

```diff
@@ -73,7 +72,9 @@ do_start()
 
   # Prepare environment
   HOME="${HOME:-$LS_HOME}"
-  JAVA_OPTS="${LS_JAVA_OPTS}"
+  # Set JAVA_OPTS to avoid sharing configuration between other
+  # JVM based programs running on the same time
+  JAVA_OPTS="-Djava.io.tmpdir=${LS_HOME}"
   ulimit -n ${LS_OPEN_FILES}
   cd "${LS_HOME}"
   export PATH HOME JAVA_OPTS LS_HEAP_SIZE LS_JAVA_OPTS LS_USE_GC_LOGGING
```

Comment:
as it should be, the PR is overriding in the lib.sh in case there is LS_JAVA_OPTS